setup_test.go

```diff
@@ -0,0 +1,461 @@
+// Copyright 2012-present Oliver Eilhard. All rights reserved.
```

Comment:
Any serious reason why you must fetch this file from original repo?

As I understand it is needed for main tests of `olivere/elastic`. But we care only about `_aggs_` files (because we copy then and modify).
Idea of copying the original tests is - to be sure that our modifications of files do not change their main behaviour and do not break them (`Source()` method).

If you do not include this file, it will make AST life easier - lots of dependency objects, structs, types are not more required

Reply:
This file contains private methods which are used in every test-file.

Reply:
aggretastic cares only about unit tests of olivere aggregations itself. Integration tests are remained on olivere side.

Every aggregation has its own unit test (the file that needs to be copied from olivere)

Two other files of aggregations use `setup_test`: `search_aggs_test.go` and `search_aggs_pipeline_test.go`, and they are not needed. First if test of `search_aggs.go` that is not imported (it's a collection of aggs) And the second is group of Integration tests for all `_pipeline_` aggs